plugins/scaffolder-backend-module-gitlab/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@backstage/plugin-scaffolder-backend-module-gitlab",
+  "version": "0.0.0",
```

Comment:
```suggestion
  "version": "1.0.0-next.1",
```

Reply:
For a new plugin this should be `0.0.0` and it will get bumped to the proper version when it gets packaged up for its first release.